samples/unit-tests/chart/parallel-coordinates-updates/demo.js

```diff
@@ -145,3 +145,71 @@ QUnit.test('Update parallel coordinates plot', function (assert) {
         'After chart update parallelInfo.counter has correct value (#10081).'
     );
 });
+
+// Calculate yAxis extremes based on the range series points as well (#15752)
+QUnit.test('Take range series points into account while calculating yAxis extremes (#15752)', function (assert) {
+    const chart = Highcharts.chart('container', {
+        chart: {
+            parallelCoordinates: true
+        },
+        series: [{
+            type: 'arearange',
+            data: [
+                [23, 25],
+                [-1, -0.5],
+                [1.5, 4],
+                [1.5, 5],
+                [4, 5]
+            ]
+        }]
+    });
+
+    const firstAxisMin = chart.yAxis[0].dataMin,
+        firstAxisMax = chart.yAxis[0].dataMax,
+        thirdAxisMin = chart.yAxis[2].dataMin,
+        thirdAxisMax = chart.yAxis[2].dataMax;
+
+    assert.deepEqual(23, firstAxisMin, '1st yAxis min should be 23.');
+    assert.deepEqual(25, firstAxisMax, '1st yAxis max should be 25.');
+
+    assert.deepEqual(1.5, thirdAxisMin, '3rd yAxis min should be 1.5.');
+    assert.deepEqual(4, thirdAxisMax, '3rd yAxis max should be 4.');
+});
+
+// Calculate plotHigh value based on each yAxis scale (#15752)
+QUnit.test('Calculate plotHigh value based on each yAxis scale (#15752)', function (assert) {
+    const chart = Highcharts.chart('container', {
+        chart: {
+            parallelCoordinates: true
+        },
+        series: [{
+            type: 'arearange',
+            data: [
+                [23, 25],
+                [-1, -0.5],
+                [1.5, 4],
+                [1.5, 5],
+                [4, 5]
+            ]
+        }]
+    });
+
+    const series = chart.series[0];
+    series.points.forEach(function (
+        point,
+        i
+    ) {
+        assert.deepEqual(
+            chart.yAxis[i].translate(
+                series.dataModify ?
+                    series.dataModify.modifyValue(point.high) : point.high,
```

Comment:
When writing tests, you should simplify the code as much as you can.

In core, we don't know whether a user is using `dataModify` or not. But here, in test, we know that we're not using it, so you can get rid of it